platforms/android/library-compose/src/main/java/io/element/android/wysiwyg/compose/internal/RichTextEditorStyleExt.kt

```diff
@@ -106,6 +118,12 @@ internal fun TextView.applyStyleInCompose(style: RichTextEditorStyle) {
     }
 }
 
+private fun TextUnitType.toTypeUnit() = when (this) {
+    TextUnitType.Sp -> TypedValue.COMPLEX_UNIT_SP
+    TextUnitType.Em -> TypedValue.COMPLEX_UNIT_FRACTION_PARENT
+    else -> TypedValue.COMPLEX_UNIT_PX
```

Comment:
```suggestion
    TextUnitType.Unspecified -> TypedValue.COMPLEX_UNIT_PX
```

Reply:
I don't think this is enough, since `TextUnitType` is a value class, not an enum. I tried using that before and the IDE forced me to add an else branch.